configuration/configuration.go

```diff
@@ -9,9 +9,10 @@ import (
 )
 
 const (
-	defaultLogLevel      = "info"
-	defaultTogglesURL    = "http://f8toggles/api"
-	defaultKeycloakURL   = "https://auth.prod-preview.openshift.io"
+	defaultLogLevel   = "info"
+	defaultTogglesURL = "http://f8toggles/api"
+	//defaultKeycloakURL   = "https://auth.prod-preview.openshift.io"
+	defaultKeycloakURL   = "https://auth.openshift.io"
```

Comment:
IMO this is confusing if you mix production URLs (keycloak) with local URLs (toggles)